var/da/da_setup_structures/da_scale_background_errors.inc

```diff
@@ -35,16 +43,55 @@ subroutine da_scale_background_errors ( be, it )
     allocate ( rf_len4(1:kz) )
     allocate ( v5_val (1:jy,1:v5_mz) )
     allocate ( rf_len5(1:1) )
+    if(cloud_cv_options >= 2 .and. cloud_cv_options .ne. 3) then
```

Comment:
For clarity, it might be nice to know which variables are mandatory for which option. This seems to be allocating everything. For example, rf_len11 is not required in the else branch at line 69

Reply:
This file is unchanged now.